components/ArgConnection.go

```diff
@@ -0,0 +1,47 @@
+package components
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/jorgerojas26/lazysql/drivers"
+	"github.com/jorgerojas26/lazysql/helpers"
+	"github.com/jorgerojas26/lazysql/models"
+)
+
+func InitFromArg(connectionString string) {
+	parsed, err := helpers.ParseConnectionString(connectionString)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Could not parse connection string: %s\n", err)
+		os.Exit(1)
+	}
```

Comment:
Please update the method signature to return an error

Calls to os.Exit should only be in main.go

Otherwise, code cannot be tested/reused